src/share/poudriere/jail.sh

```diff
@@ -907,6 +936,7 @@ saved_argv="$@"
 shift $((OPTIND-1))
 
 METHOD=${METHOD:-ftp}
+CLEAN=${CLEAN:-none}
```

Comment:
`CLEAN` defaults to `0` now in common.sh. I suggest renaming this field to CLEANJAIL or just `unset CLEAN` near `XDEV=0` above.